frontend/plan/components/search/ButtonFilter.tsx

```diff
@@ -43,11 +43,21 @@ export function FilterButton<
             setIsActive(false);
         } else {
             setIsActive(true);
-            updateButtonFilter(activeSchedule);
-            startSearch({
-                ...filterData,
-                [buttonProperty]: {[activeSchedule]: null},
-            });
+            if (buttonProperty === "fit_schedule") {
+                updateButtonFilter(value);
+                startSearch({
+                    ...filterData,
+                    [buttonProperty]: value,
+                });
+            } 
+            if (buttonProperty === "is_open") {
+                updateButtonFilter(true);
+                startSearch({
+                    ...filterData,
+                    [buttonProperty]: true,
+                });
+            } 
+            
         }
```

Comment:
Can we generalize the value so we don't have to handle different buttonProperty, maybe lets change is_open to take number like 0 or -1 for false and 1 for true. This way we don't need both if statement and remove `having number | boolean` type